src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -137,7 +137,6 @@ public static void DigestAuthentication_RoundTrips_Echo()
 
     [Fact]
     [OuterLoop]
-    [ActiveIssue(604, PlatformID.AnyUnix)]
     public static void NtlmAuthentication_RoundTrips_Echo()
```

Comment:
I'm not sure how the NTLM authentication code works on *NIX. If there's a concept of being logged in, there might be some default credentials which could work. @stephentoub , as you've done some work with the libcurl implementation of HttpClientHandler, do you know the answer to this?